### ISSUE TITLE
[0.21.6] Can't even compile examples

### ISSUE THREAD

**Issue by keltia** [rustls/rustls#1397]
Did the examples lag behind the actual library code?

Trying the compile the simpliest examples ine simpleclient or the mio one leads to this:
```text
error[E0433]: failed to resolve: could not find `crypto` in `rustls`
 --> sources/examples/tls.rs:5:13
  |
5 | use rustls::crypto::ring::Ring;
  |             ^^^^^^ could not find `crypto` in `rustls`

error[E0432]: unresolved import `rustls::crypto`
 --> sources/examples/tls.rs:6:13
  |
6 | use rustls::crypto::CryptoProvider;
  |             ^^^^^^ could not find `crypto` in `rustls`
```
Looking at the source code, everything seems to be there.

It can't find the `webpki_roots` dep either.

Thanks.

**Comment by ctz** [comment]
For 0.21.6 please take the example code from that version: https://github.com/rustls/rustls/tree/v/0.21.6 rather than main.

**Comment by cpu** [comment]
I think this issue can be closed The examples from the 0.21.6 tree do compile as expected.

If you run into any additional trouble please feel free to open a new issue, or comment here.

Thanks!
